test/test_oneview_server_profile.py

```diff
@@ -700,8 +700,8 @@ def test_should_fail_when_network_not_found_on_creation(self):
         self.mock_ansible_module.fail_json.assert_called_once_with(msg=expected_error)
 
     def test_should_replace_server_hardware_type_name_by_uri(self):
-        sht_uri = "/rest/server-hardware-types/BCAB376E-DA2E-450D-B053-0A9AE7E5114C"
-        sht = {"name": "SY 480 Gen9 1", "uri": sht_uri}
+        SERVER_HARDWARE_TEMPLATE_URI = "/rest/server-hardware-types/BCAB376E-DA2E-450D-B053-0A9AE7E5114C"
+        sht = {"name": "SY 480 Gen9 1", "uri": SERVER_HARDWARE_TEMPLATE_URI}
```

Comment:
But here the sht_uri's were humble variables, and not the _constants_ defined in the test header